packages/lesswrong/components/walledGarden/GatherTown.tsx

```diff
@@ -175,8 +176,7 @@ const GatherTown = ({classes}: {
           No users currently online. Check back later or be the first to join!
           {tooltip}
         </div>}
-        <GardenCodesList terms={{view:"semipublicGardenCodes", types: eventTypes}} />
-        <a className={classes.allEvents} href={`https://calendar.google.com/calendar/u/0?cid=${CAL_ID}`}>View All Events</a>
+        {!isMobile() && <GardenCodesList limit={2}/>}
```

Comment:
Hmm, this might very well break. Can someone quickly test this during SSR?

Reply:
Oh, sorry that got lost in the shuffle during the transfer to Ruby. I'll do a quick PR switching it to a JSS clause

Reply:
The terms keyword shouldn't be there. That was taken out in my PR.

Reply:
(deleted comment, sorry, was being dumb)

Can you explain the SSR issue and how to check it?

Reply:
Ray is going to switch it to a CSS thing, so won't matter

Reply:
Yeah, but I still want to understand SSR issues in general.

Reply:
When you load a URL directly (as opposed to navigating within-site to that URL), it's rendered first on the server, and again on the client. Every component that's in the tree for that render needs to render the same way both times, or else bad things happen. In particular, that means it can't depend on any information which is only available in the browser, which includes screen width and the sizes of things in general. Since `isMobile` depends on information that's only available in the browser, it can't be safely used during SSR.

(Unfortunately `isMobile` just returns false when used in SSR, which makes it a footgun.)